src/couch_replicator_scheduler.erl

```diff
@@ -30,13 +30,14 @@
 -export([handle_config_change/5, handle_config_terminate/3]).
 
 %% types
--type event_type() :: started | stopped | crashed.
+-type event_type() :: started | stopped | {crashed, any()}.
```

Comment:
where is the Reason term used? since it'll often be large, and we'll have lots of jobs in memory I think we might not want to store it.